src/main/java/org/testng/TestNG.java

```diff
@@ -821,7 +822,9 @@ public void addInvokedMethodListener(IInvokedMethodListener listener) {
   }
 
   public Set<IReporter> getReporters() {
-    return m_reporters;
+    //This will now cause a different behavior for consumers of this method because unlike before they are no longer
```

Comment:
I'm not sure the comment has some added value because once merged, it is done. But the information could be place in the commit message instead.